Makefile

```diff
@@ -35,3 +35,6 @@ ocp_cleanup:
 
 host_cleanup:
 	./host_cleanup.sh
+
+bell:
+	echo "Done!" $$'\a'
```

Comment:
Trivial suggestion ... I'd add a '@' prefix to your echo command, which silences the output of the command itself so you get:

```
$ make bell
Done!
```

instead of:

```
$ make bell
echo "Done!" $'\a'
Done! 
```

Reply:
Done